src/Illuminate/Translation/Translator.php

```diff
@@ -147,7 +162,13 @@ public function get($key, array $replace = [], $locale = null, $fallback = true)
         // If the line doesn't exist, we will return back the key which was requested as
         // that will be quick to spot in the UI if language keys are wrong or missing
         // from the application's language files. Otherwise we can return the line.
-        return $this->makeReplacements($line ?: $key, $replace);
+        $line = $this->makeReplacements($line ?: $key, $replace);
+
+        foreach ($this->missingTranslationHandlers as $missingTranslationHandler) {
+            $missingTranslationHandler($key);
```

Comment:
@nunomaduro Should we use `call_user_func()` here (I noticed that `call_user_func()` in a few different other places as well) or is this OK?

Reply:
This is fine.